src/npc.cpp

```diff
@@ -26,13 +26,13 @@ void Npcs::reload()
 	}
 }
 
-Npc* Npc::createNpc(const std::string& name)
+std::shared_ptr<Npc> Npc::createNpc(const std::string& name)
 {
-	std::unique_ptr<Npc> npc(new Npc(name));
+	auto npc = std::make_shared<Npc>(name);
```

Comment:
Could this be `const auto`
```suggestion
	const auto npc = std::make_shared<Npc>(name);
```